### PR TITLE
Added basic Ward flags to Card

### PR DESCRIPTION
**Pull request by romanogit**
Added basic Ward flags to Card and enough to allow cards to use Ward actions
Merge after Jeremy's latest PRs should be minimal.

### CODE REVIEW

File: less/cards.less [hunk 1]
```diff
@@ -429,6 +429,26 @@
   color: transparent;
 }
 
+.enrage {
+  background-image: url("/img/enrage.png");
+  line-height: 21px;
+}
+
+.enrage.single {
```
Comment (cryogen) [inline less/cards.less]: This is duplicated so should go in a single “.single” class. But, more than that, this whole hiding should be done in the jsx not the css

File: server/game/cards/03-WC/ForceField.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    setupCardAbilities(ability) {
+        this.whileAttached({
+            effect: ability.effects.gainAbility('reap', {
+                gameAction: ability.actions.ward()
+            })
+        });
+    }
+}
+
+ForceField.id = 'force-field'; // This is a guess at what the id might be - please check it!!!
```
Comment (cryogen) [inline server/game/cards/03-WC/ForceField.js]: Please remove the comment

File: server/game/cards/03-WC/TricerianLegionary.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+        this.play({
+            target: {
+                cardType: 'creature',
+                controller: 'self',
+                gameAction: ability.actions.ward()
+            }
+        });
+    }
+}
+
```
Comment (cryogen) [inline server/game/cards/03-WC/TricerianLegionary.js]: Please remove the comment

File: less/cards.less (hunk 1 above)
Comment (romanogit) [reply less/cards.less]: what do you mean in JSX?
JSX should hold the logic of not showing the number?

Comment (romanogit) [reply less/cards.less]: Fixed

**Pull request merged**